src/databricks/labs/remorph/reconcile/connectors/snowflake.py

```diff
@@ -1,28 +1,43 @@
 import re
 
+from databricks.labs.blueprint.entrypoint import get_logger
 from pyspark.errors import PySparkException
 from pyspark.sql import DataFrame
 from pyspark.sql.functions import col
 
+# pylint: disable=ungrouped-imports
 from databricks.labs.remorph.reconcile.connectors.data_source import DataSource
 from databricks.labs.remorph.reconcile.constants import SourceDriver, SourceType
-from databricks.labs.remorph.reconcile.recon_config import JdbcReaderOptions, Schema
+from databricks.labs.remorph.reconcile.recon_config import (
+    JdbcReaderOptions,
+    Schema,
+    Table,
+    TableRecon,
+)
+from databricks.labs.blueprint.entrypoint import get_logger
+
+logger = get_logger(__file__)
+
+logger = get_logger(__file__)
 
 
 class SnowflakeDataSource(DataSource):
     @property
     def get_jdbc_url(self) -> str:
-        return (
+        jdbc_url = (
             f"jdbc:{SourceType.SNOWFLAKE.value}://{self._get_secrets('account')}.snowflakecomputing.com"
             f"/?user={self._get_secrets('sfUser')}&password={self._get_secrets('sfPassword')}"
             f"&db={self._get_secrets('sfDatabase')}&schema={self._get_secrets('sfSchema')}"
             f"&warehouse={self._get_secrets('sfWarehouse')}&role={self._get_secrets('sfRole')}"
         )
+        logger.debug(f"JDBC URL: {jdbc_url}")
```

Comment:
property shouldn't have loggers, logger should be implemented outside where the property is called.